projects/1.18/assets/ars-elemental/ars_elemental/lang/zh_cn.json

```
@@ -0,0 +1,147 @@
+  "block.ars_elemental.water_upstream": "翻腾之心",
+  "block.ars_elemental.mermaid_rock": "塞壬神坛",
+  "block.ars_elemental.yellow_archwood": "闪耀至高木木头",
+  "block.ars_elemental.yellow_archwood_log": "闪耀至高木原木",
+  "block.ars_elemental.stripped_yellow_archwood": "去皮闪耀至高木原木",
+  "block.ars_elemental.stripped_yellow_archwood_log": "去皮闪耀至高木原木",
+  "block.ars_elemental.yellow_archwood_sapling": "闪耀至高木树苗",
+  "block.ars_elemental.yellow_archwood_leaves": "闪耀至高木树叶",
+  "block.ars_elemental.spore_blossom_up": "孢子花（地面）",
+  "block.ars_elemental.fire_turret": "火之灌注炮台",
+  "block.ars_elemental.water_turret": "水之灌注炮台",
+  "block.ars_elemental.air_turret": "气之灌注炮台",
+  "block.ars_elemental.earth_turret": "土之灌注炮台",
+  "entity.ars_elemental.summon_skelehorse": "被召唤的骷髅马",
+  "entity.ars_elemental.summon_direwolf": "被召唤的冰原狼",
+  "entity.ars_elemental.summon_vhex": "被召唤的邪恶恼鬼",
+  "entity.ars_elemental.summon_wskeleton": "被召唤的复仇者",
+  "entity.ars_elemental.summon_dolphin": "被召唤的海豚",
+  "entity.ars_elemental.summon_strider": "被召唤的炽足兽",
+  "entity.ars_elemental.summon_mk": "Summoned Emmekiappo",
+  "entity.ars_elemental.linger_magnet": "引力场",
+  "entity.ars_elemental.siren_entity": "塞壬",
+  "entity.ars_elemental.siren_familiar": "塞壬",
+  "entity.ars_elemental.firenando_entity": "烈焰加农",
+  "entity.ars_elemental.firenando_familiar": "烈焰加农",
+  "entity.ars_elemental.fire_mage": "漂泊火之法师",
+  "entity.ars_elemental.water_mage": "漂泊水之法师",
+  "entity.ars_elemental.air_mage": "漂泊气之法师",
+  "entity.ars_elemental.earth_mage": "漂泊土之法师",
+  "entity.ars_elemental.flashing_weald_walker": "闪耀林地行者",
+  "key.ars_elemental.open_pouch": "[元素魔艺] 打开小饰品袋",
+  "curios.modifiers.an_focus": "当放置在法术核心槽时:",
+  "curios.modifiers.bangle": "当放置在手环槽时:",
+  "curios.identifier.bundle": "手袋",
+  "curios.identifier.bangle": "手环",
+  "effect.ars_elemental.watery_grave": "沉重之水",
+  "effect.ars_elemental.watery_grave.description": "使目标更快沉入水底并加速氧气消耗。",
+  "effect.ars_elemental.hellfire": "地狱之火",
+  "effect.ars_elemental.hellfire.description": "地狱之火可以让下界生物燃烧，降低其治疗效果，并使其更容易被火焰伤害。",
+  "effect.ars_elemental.life_link": "生命链接",
+  "effect.ars_elemental.life_link.description": "施法者将会把自身受到的伤害共享给目标，目标会把受到的治疗效果共享给施法者。",
+  "effect.ars_elemental.enthralled": "魂不着体",
+  "effect.ars_elemental.enthralled.description": "使目标生物被迷惑，令其不会攻击施法者，而是为施法者战斗。",
+  "effect.ars_elemental.hymn_of_order": "秩序之歌",
+  "effect.ars_elemental.hymn_of_order.description": "这是一个无法获得的效果，仅供管理员使用，它会使目标玩家的法力值设置为1，并禁用目标玩家施法，从而让他们无法利用魔法在服务器中搞破坏。",
+  "enchantment.ars_elemental.mirror_shield": "法术镜射",
+  "enchantment.ars_elemental.mirror_shield.desc": "将其附魔在巫师之盾上，让它可以像镜子一样反弹法术。",
+  "ars_nouveau.page.elemental_tweaks": "元素魔艺带来的新特性",
+  "ars_elemental.page.elemental_tweaks": "当你安装了元素魔艺, 以下特性将会生效:$(br)巫师之盾可以在阻挡时触发反应魔咒。$(br)被召唤的闪电将不会摧毁物品。$(br)如果粉碎构件+敏感构件的组合可以处理掉落物。$(br)寒流构件会对冰冻的生物造成更多伤害。$(br)点燃构件能够将冰融化成水。",
+  "ars_elemental.page1.flashing_archwood": "这种逸散着金光的树与天空有着密切的关系。像其他的至高木一样，会在任何生物群系小概率生成。它也同样可以用来制作仪式之板。它对应的林地行者会对敌人使用发射将其抛掷到高空，并在敌人处于空中时对其使用风切。",

```

```suggestion
  "ars_elemental.page1.flashing_archwood": "这种逸散着金光的树与天空关系密切。与其他的至高木一样，在所有生物群系小概率生成。可用于制作仪式之板。其对应的林地行者将敌人发射至高空，并对敌人使用风切。",
```
[翻译腔 - Wikipedia](https://zh.wikipedia.org/wiki/Wikipedia:%E7%BF%BB%E8%AF%91%E8%85%94)